evalml/model_understanding/graphs.py

```diff
@@ -569,6 +572,90 @@ def graph_prediction_vs_actual(y_true, y_pred, outlier_threshold=None):
     return _go.Figure(layout=layout, data=data)
 
 
+def get_tree_data(clf):
+    """Return data for a fitted tree in a restructured format
+
+    Arguments:
+        clf (ComponentBase): A fitted tree-based estimator.
+
+    Returns:
+        OrderedDict: An OrderedDict of OrderedDicts describing a tree structure
+    """
+    if not clf.model_family == ModelFamily.DECISION_TREE:
+        raise ValueError("Tree structure reformatting is not supported for non-Tree estimators")
```

Comment:
Really nit-picky, but just to clarify, this only works for decision trees right? That is, RF or Extra Trees doesn't work? If that's true, can we update the docstring / error to say that?